lib/system/excpt.nim

```diff
@@ -229,16 +229,20 @@ proc auxWriteStackTrace(f: PFrame; s: var seq[StackTraceEntry]) =
   while it != nil:
     s[last] = StackTraceEntry(procname: it.procname,
                               line: it.line,
+                              col: it.col,
                               filename: it.filename)
     it = it.prev
     dec last
 
-template addFrameEntry(s, f: untyped) =
+template addFrameEntry(s: var string, f: PFrame|StackTraceEntry) =
   var oldLen = s.len
   add(s, f.filename)
   if f.line > 0:
     add(s, '(')
+    # todo: reuse `lineInfoToString`
```

Comment:
note to reviewer: ok to add this dependency?

Reply:
Probably.

Reply:
replaced w lineInfoToString (system.nim already included it via helpers.nim so no new dependency added)